schemas/table-of-contents/table-of-contents.json

```diff
@@ -1,109 +1,102 @@
 {
-  "$schema":"http://json-schema.org/draft-07/schema#",
-  "definitions":{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://github.com/CMSgov/price-transparency-guide/blob/master/schemas/table-of-contents/table-of-contents.json",
+  "definitions": {
     "file_location": {
       "type": "object",
       "properties": {
         "description": {
-          "type": "string"
+          "type": "string",
+          "minLength": 1
         },
         "location": {
```

Comment:
I would also like to just get rid of this location lookup component.

They can hide things by not publishing or breaking the links later.   Also are they setup to handle an deluge of web requests to resolve.

It makes the files not stand on their own.  you have to also parse them within the payer time window vs just downloading them.    (cigna is the big user of this)

Reply:
Removing the `location` from the table of contents would be pretty fundamental in how multiple plans can be associated with shared negotiated rates. It is one of the sources of a pretty large reduction of duplicative data. We are going to keep it for the time being.